src/darwin/Framework/CHIP/CHIPDeviceController_Internal.h

```diff
@@ -79,6 +79,11 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (CHIP_ERROR)isRunningOnFabric:(chip::FabricInfo *)fabric isRunning:(BOOL *)isRunning;
 
+/**
+ * Temporary wrapper for adding a group with all the necessary informations (e.g key sets).
```

Comment:
It's not just adding a group.  It's adding a local endpoint to the group....

I'm glad this is _Internal, but this really needs to not exist, ideally.